nucleus/validate/scenario_test.py

```diff
@@ -224,6 +244,7 @@ def upload_external_evaluation_results(
             "overall_metric": aggregate_weighted_sum / aggregate_weight,
             "model_id": model_id,
             "slice_id": self.slice_id,
+            "level": str(level),
```

Comment:
Do multiple inheritance for EntityLevel from `str` and `Enum` and just pass the enum